.flake8

```diff
@@ -1,5 +1,5 @@
 [flake8]
-ignore = E203, E266, E501, W503, E123, C901
+ignore = E203, E266, E501, W503, E123, C901, W504
```

Comment:
Can we use a `# noqa: W504` comment on the troublesome lines rather than add it to the ignore list?